sdk/search/Microsoft.Azure.Search/tests/Tests/DataSourceTests.cs

```diff
@@ -17,13 +17,13 @@ public sealed class DataSourceTests : SearchTestBase<SearchServiceFixture>
     {
         private const string FakeDescription = "Some data source";
 
-        [Fact]
+        [Fact(Skip = "Needs AzureSqlReadOnlyConnectionString to be set")]
```

Comment:
We also need to delete the corresponding recordings.  Anything that shows up in https://github.com/Azure/azure-sdk-for-net/search?q=azs-playground should go.

//fyi @brjohnstmsft, but I don't think that'll be a problem since we're no longer updating the old libraries